main.go

```diff
@@ -7,12 +7,15 @@ import (
 	"gopkg.in/alecthomas/kingpin.v2"
 
 	"github.com/gunk/gunk/convert"
+	"github.com/gunk/gunk/debug"
 	"github.com/gunk/gunk/format"
 	"github.com/gunk/gunk/generate"
 )
 
 var (
-	app = kingpin.New("gunk", "Gunk Unified N-terface Kompiler command-line tool.")
+	app                   = kingpin.New("gunk", "Gunk Unified N-terface Kompiler command-line tool.")
+	debugCommands         = app.Flag("print-commands", "print the commands").Short('x').Bool()
+	debugPackageGenerated = app.Flag("print-package-generated", "print the names of packages as they are generated").Short('v').Bool()
```

Comment:
I think these should be on a per-command basis, much like config loading. For example, why should `gunk format -x` make sense, given that it's never going to run any external program?

Similarly, we want `gunk generate -v -x`, not `gunk -v -x generate`. Particularly since it's very unusual for Go programas to take any arguments after flags.

Reply:
Done.